packages/button/src/LionButton.js

```diff
@@ -98,41 +94,36 @@ export class LionButton extends DisabledWithTabIndexMixin(SlotMixin(LitElement))
           border: 0; /* reset default agent styles */
         }
 
-        .click-area {
-          position: absolute;
-          top: 0;
-          right: 0;
-          bottom: 0;
-          left: 0;
-          margin: 0;
-          padding: 0;
-        }
-
-        :host(:focus:not([disabled])) .btn {
+        /* Show focus styles on keyboard focus. */
+        :host(:focus:not([disabled])),
+        :host(:focus-visible) {
```

Comment:
This is a progressive enhancement for browsers that support it, but does not break anything for browsers that don't support this selector, correct?

Reply:
Correct. It works as before with `:host(:focus:not([disabled]))` and it will hide the focus for `:host(:focus:not(:focus-visible))` if it is supported.

Chrome 86 support it. Previous versions works fine.